src/format/locales.rs

```diff
@@ -0,0 +1,361 @@
+use super::{FormatError, FormatErrorKind, FormatResult};
+
+macro_rules! locale_match {
```

Comment:
I think it would be nice if this function lived in pure-rust-locales instead of here.

Reply:
Done